zcash/client.go

```diff
@@ -162,6 +168,15 @@ func newSeederPeerConfig(magic network.Network, template *peer.Config) (*peer.Co
 	}
 	newPeerConfig.ChainParams = params
 
+	switch magic {
+	case network.Mainnet, network.Regtest:
```

Comment:
This is just a note for my own understanding:

I think we'd use the mainnet version for regtest before mainnet activation, and the testnet version for regtest before testnet activation.

Since we don't use our seeder for regtest, and Zebra doesn't support regtest, we don't need to change anything.